src/device/reduce_kernel.h

```diff
@@ -254,6 +261,30 @@ SPECIALIZE_REDUCE(FuncMinMax, half, 1, half, fn.isMinNotMax ? __hmin(x, y) : __h
 #endif
 #endif
 
+#if defined(RCCL_FLOAT8)
+#if __CUDA_ARCH__ >= 800
```

Comment:
We can remove this branch directly? There is no plan to support running RCCL on Nvidia hardware.

Reply:
removed